crt_portal/cts_forms/templates/forms/confirmation.html

```diff
@@ -94,60 +93,61 @@ <h3 class="intake-section-title">Our specialists determine the next step</h3>
               3
             </div>
             <h3 class="intake-section-title">
-                When possible, we will follow up with you
+                {% trans "When possible, we will follow up with you" %}
             </h3>
           </div>
           <div class='confirmation-content padding-left-6'>
-            We do our best to let you know about the outcome of our review. However, we may not always be able to provide you with updates because:
+            {% trans "We do our best to let you know about the outcome of our review. However, we may not always be able to provide you with updates because:" %}
             <ul class="confirmation-list">
-              <li
-                >We’re actively working on an investigation or case related to your report.
+              <li>
+                {% trans "We’re actively working on an investigation or case related to your report." %}
               </li>
               <li>
-                We’re receiving and actively reviewing many requests at the same time.
+                {% trans "We’re receiving and actively reviewing many requests at the same time." %}
               </li>
             </ul>
-            <p><strong>If we are able to respond, we will contact you within 4 weeks using the contact information you provided</strong> in this report.</p>
+            <p>{% trans "<strong>If we are able to respond, we will contact you within 4 weeks using the contact information you provided</strong> in this report." %}</p>
           </div>
         </li>
       </ol>
       <hr>
     </div>
 
     <div class="tablet:grid-col-8 tablet:grid-offset-2">
-      <h2>What you can do next</h2>
+      <h2>{% trans "What you can do next" %}</h2>
       <ol class='confirmation-list'>
         <li>
           <div class="title">
             <div class="title-icon-blue margin-right-2">
-              1
+              {% trans "1" %}
```

Comment:
Should these be translated? If yes, there's a few missing template tags int the section numbering above.

Reply:
This PR is for Spanish, so the numbers are going to be the same. I can take them out for now if it is confusing.

I do believe that we will have to start adding the numbers for other languages- but haven't confirmed. I tried to find the hard coded numbers, since that was low hanging fruit but there are some calculated numbers that we will have to do some refactoring for in the future. 

For the epic: https://app.zenhub.com/workspaces/doj-crt-intake-scrum-board-5d03bf56c1c8a35d482eca0f/issues/18f/crt-portal/201
The eventual languages will be:
- Spanish
- Korean
- Taglog
- Vietnamese
- Mandarin
- Simplified Chinese